src/registry_schemas/schemas/incorporation.json

```diff
@@ -43,7 +44,34 @@
         },
         "incorporationAgreement": {
             "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/agreementType"
+        },
+        "hasNameTranslations": {
```

Comment:
The Create UI declares IncorporationFilingIF, which should be updated to include this new boolean and array.

Reply:
@cameron-freshworks is working on that